tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -579,15 +581,18 @@ export class RunsTableContainer implements OnInit, OnDestroy {
     return combineLatest([
       this.store.select(getRuns, {experimentId}),
       this.store.select(getRunColorMap),
+      this.store.select(getCurrentRouteRunSelection),
       this.runsColumns$,
       this.runToHParamValues$,
     ]).pipe(
-      map(([runs, colorMap, runsColumns, runToHParamValues]) => {
+      map(([runs, colorMap, selectionMap, runsColumns, runToHParamValues]) => {
         return runs.map((run) => {
           const tableData: TableData = {
             id: run.id,
             color: colorMap[run.id],
+            selected: (selectionMap && selectionMap.get(run.id)) ?? false,
```

Comment:
I'm only like 80% this is correct, but I think this can be streamlined a bit.

```suggestion
            selected: Boolean(selectionMap?.get(run.id)),
```